Feature/update documentation (#92)
cloudogu/usermgt: docs/gui/usermanagement_en.md

You can't save the password until all password policies have been satisfied.

If all password policies are fulfilled, the field will be marked green and the new password can be saved.

![All rules satisfied](figures/usermanagement/CESUsermanagement_Password_Policy_All_Rules_Satisfied_en.png)

## Administration of groups

As an administrator, the **User Management** gives you the ability to **create, edit, or delete groups, as well as manage the members of groups**.

Groups can be used to manage different permission configurations for different user groups. More information about this can be found in the section [Permission concept in the Cloudogu EcoSystem](#permission-concept-in-the-cloudogu-ecosystem).

### System groups

In the **User Management** you will find two predefined groups. Members of these groups have special permissions in the Cloudogu EcoSystem Dogus.

**Manager Group**

Members of the **Manager Group** have **full access to the User Management** of the Cloudogu EcoSystem.
This gives users the authorization to create and manage additional users and groups.
Beyond that, no other permissions are associated with the *Manager Group**.

> You can change the *Manager group* to be used by entering the following in the configuration of the Cloudogu EcoSystem in the entry 

Reviewer: ```suggestion
You can change the *Manager group* to be used by changing the entry `/config/_global/manager_group` in the configuration of the Cloudogu EcoSystem to the desired group: 
```

`/config/_global/manager_group` to set the desired group:

```shell
etcdctl set /config/_global/manager_group newManagerGroup
```

> The **User Management** Dogu must then be restarted for the change to take effect.

Reviewer: ```suggestion
The **User Management** Dogu needs to be restarted to take the change into account.
```

**Admin Group**

Members of this group have administrative rights in **all** Dogus of the Cloudogu EcoSystem. 
These users can use the administrative functions in the individual Dogus and thus, for example, install plug-ins or make application settings.

> The **Backup & Restore** Dogu contains only administrative functions and does not use a separate role concept.

Reviewer: ```suggestion
The usage of the **Backup & Restore** Dogu is restricted to the administrators.
```

Consequently, only users who are members of the **Admin group** have access to the **Backup & Restore** Dogu.

You can change the **Admin group** to be used by going to the configuration of the Cloudogu EcoSystem in the entry
`/config/_global/admin_group` to set the desired group:

```shell
etcdctl set /config/_global/admin_group newAdminGroup
```

> All Dogus must then be restarted for the change to take effect.

Reviewer: ```suggestion
All Dogus needs to be restarted for the change to take effect.
```

### Creating a new group

> To avoid having to assign individual permissions for all accounts in the Dogus, you can create groups as described below:

Reviewer: ```suggestion
To avoid assigning individual permissions for all accounts in the Dogus, you can create groups as described below:
```

System groups **cannot** be deleted.

Note that groups deleted in the **User Management** are not automatically deleted in the Dogus as well. However, the assignment of accounts to groups is synchronized.

## Permission concept in the Cloudogu EcoSystem

The permission concept of the Cloudogu EcoSystem is based on a **central user management** and a **decentralized permission configuration**: Accounts and groups can be stored in the **User Management**. These are propagated to the other Dogus of the Cloudogu EcoSystem, allowing you to assign permissions for groups or individual users decentralized in each Dogu.

### User Management

The **User Management** is used to manage accounts and groups. In doing so, the **User Management** uses an internal *LDAP* as directory service.

In addition to using the **User Management** provided by us, you have the option of using an **external directory service** for the Cloudogu EcoSystem. In this case, user management would **not** be done via the Dogu **User Management** presented here, but via the external directory service you have connected (such as an external *LDAP* or *Active Directory*).

### Permission concepts of the Dogus

> You can create the accounts for users of the Cloudogu EcoSystem centrally in the **User Management**. To simplify the permission configuration, you can create groups for different user groups. A user can belong to more than one group. A group can have more than one member.

Reviewer: ```suggestion
You can create the accounts for users of the Cloudogu EcoSystem centrally in the **User Management**. To simplify the permission configuration, you can create groups for different purposes. A user can belong to more than one group. A group can have more than one member.
```

Accounts and groups are **synchronized** with the Dogus, meaning in each Dogu you will find the accounts and groups created in the **User Management**. 

> Since Dogus may be systems developed outside the Cloudogu EcoSystem, the **permission concept in the Dogus may differ** - as you can see in the following diagram exemplary.

Reviewer: ```suggestion
Since Dogus may be applications developed outside of the Cloudogu EcoSystem, the **permission concept in the Dogus may differ** - as you can see in the following diagram exemplary.
```

![Rights concept in the Dogus](figures/usermanagement/RoleConceptCloudoguEcoSystem_en.png)

For more information about the permission concept of individual Dogus, please refer to the documentation of the respective Dogu.

You can also create accounts and groups directly in the Dogus. Please note that these are then not known to other Dogus and cannot be managed in the **User Management**. Creating accounts or groups outside of the **User Management** is therefore **not** recommended.

### Synchronization of accounts and groups

> User accounts and groups are passed on to a Dogu **as soon as the user with the respective account calls up the Dogu**. For this purpose, the user's group assignments are queried every time the user logs on to a Dogu via the CAS (Central Authentication Service) - the central *single sign-on* authentication service of the Cloudogu EcoSystem.

Reviewer: logs on to - logs into?
